src/utils/base.test.js

```diff
@@ -0,0 +1,221 @@
+import { getStyles } from './base';
+/* eslint-disable */
```

Comment:
So this will disable the entirety of ESLint for the file. We want to avoid doing this so that if future contributors alter this file they won't unintentionally be violating our ESLint setup. Prefer to always use `eslint-disable-line` or `eslint-disable-next-line` with the specific rule. In this case, it looks like you're violating the `max-statements` rule, which is totally fine for a test file. So just replace `/* eslint-disable */` with `eslint-disable-next-line max-statements` here.